Stacks/ValidateParaentheses.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections;
+
+ class ValidatingParaentheses{
+  public static void Main(string[] args){
+    Console.WriteLine(validate(""));
+  }
+
+  private static bool validate(string str){
+    Stack s = new Stack();
+      for(int i=0; i<str.Length; i++){
+        if(str[i] == '('){
+          s.Push(str[i]);
+        }
+        if(str[i] == ')'){
+          try{
+            s.Pop();
+          }catch(Exception e){
```

Comment:
Add a comment to signify what this Exception means, change this to specific exception that we would want to catch